spec/factories/user.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :user do
+    first_name "Steve"
+    last_name  "Jones"
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.